css/css-scroll-snap/scroll-target-align-003.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and snap position with snapping off (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#choosing'>
+<link rel='match' href='scroll-target-001-ref.html'>
+<meta name="flags" content="may">
```

Comment:
An assert meta would be welcome, similarly to scroll-target-001

Reply:
Done.